### DISCUSSION TITLE
Admin page 500 error after updating to 1.7.8.8 - DBALException

### DISCUSSION THREAD

**Discussion by jpcoke** [PrestaShop/PrestaShop#30935] [Q&A] [answered]
Hello 

As mentioned i did upgrade from 1.7.8.7 para 1.7.8.8 via 1 clickupgrade  and cant acess admin page getting a 500 error.

By activating ebug mode i get:

(4/4) DBALException

An exception occured while establishing a connection to figure out your platform version.
You can circumvent this by setting a 'server_version' configuration value

For further information have a look at:
https://github.com/doctrine/DoctrineBundle/issues/673

in ConnectionFactory.php line 79

at ConnectionFactory->getDatabasePlatform(object(Connection))in ConnectionFactory.php line 50

at ConnectionFactory->createConnection(array('driver' => 'pdo_mysql', 'host' => 'xxxx', 'port' => '', 'dbname' => 'xxxxxxxx', 'user' => 'xxxxxxx', 'password' => 'xxxxxxxx', 'charset' => 'utf8mb4', 'driverOptions' => array('SET sql_mode=(SELECT REPLACE(@@sql_mode,\'ONLY_FULL_GROUP_BY\',\'\'))', false), 'defaultTableOptions' => array()), object(Configuration), object(ContainerAwareEventManager), array('enum' => 'string'))in appDevDebugProjectContainer.php line 2820

at appDevDebugProjectContainer->getDoctrine_Dbal_DefaultConnectionService()in appDevDebugProjectContainer.php line 2858

at appDevDebugProjectContainer->getDoctrine_Orm_DefaultEntityManagerService()in getPrestashop_Translation_DatabaseLoaderService.php line 8

at require('/home/xxxxxx/public_html/xxxxxxx.com/var/cache/dev/ContainerXpgdbtw/getPrestashop_Translation_DatabaseLoaderService.php')in appDevDebugProjectContainer.php line 2747

at appDevDebugProjectContainer->load('getPrestashop_Translation_DatabaseLoaderService.php')in appDevDebugProjectContainer.php line 4982

at appDevDebugProjectContainer->ContainerXpgdbtw\{closure}()in ServiceLocator.php line 64

at ServiceLocator->get('prestashop.translation.database_loader')in Translator.php line 141

at Translator->initialize()in Translator.php line 127

at Translator->initializeCatalogue('pt-PT')in Translator.php line 314

at Translator->dumpCatalogue('pt-PT', object(ResourceCheckerConfigCache))in Translator.php line 299

at Translator->Symfony\Component\Translation\{closure}(object(ResourceCheckerConfigCache))in ResourceCheckerConfigCacheFactory.php line 43

at ResourceCheckerConfigCacheFactory->cache('/home/xxxxx/public_html/xxxxxxxx.com/var/cache/dev/translations/catalogue.pt-PT.L8dqxxF.php', object(Closure))in Translator.php line 300

at Translator->initializeCacheCatalogue('pt-PT')in Translator.php line 265

at Translator->loadCatalogue('pt-PT')in Translator.php line 241

at Translator->getCatalogue('pt-PT')in Translator.php line 198

at Translator->trans('Successful deletion.', array(), 'AdminNotificationsSuccess', null)in PrestaShopTranslatorTrait.php line 61

at Translator->trans('Successful deletion.', array(), 'AdminNotificationsSuccess', null)in LoggingTranslator.php line 47

at LoggingTranslator->trans('Successful deletion.', array(), 'AdminNotificationsSuccess', null)in DataCollectorTranslator.php line 50

at DataCollectorTranslator->trans('Successful deletion.', array(), 'AdminNotificationsSuccess', null)in PrestaShopTranslatorTrait.php line 61

at DataCollectorTranslator->trans('Successful deletion.', array(), 'Admin.Notifications.Success', null)in Controller.php line 338

at ControllerCore->trans('Successful deletion.', array('legacy' => 'htmlspecialchars'), 'Admin.Notifications.Success')in AdminController.php line 481

at AdminControllerCore->__construct()in LegacyContext.php line 86

at LegacyContext->getContext()in UserLocaleListener.php line 39

at UserLocaleListener->__construct(object(LegacyContext))in appDevDebugProjectContainer.php line 3554

at appDevDebugProjectContainer->getPrestashop_UserLocale_ListenerService()in appDevDebugProjectContainer.php line 4083

at appDevDebugProjectContainer->ContainerXpgdbtw\{closure}()in EventDispatcher.php line 231

at EventDispatcher->sortListeners('kernel.request')in EventDispatcher.php line 61

at EventDispatcher->getListeners('kernel.request')in ContainerAwareEventDispatcher.php line 129

at ContainerAwareEventDispatcher->getListeners('kernel.request')in TraceableEventDispatcher.php line 259

at TraceableEventDispatcher->preProcess('kernel.request')in TraceableEventDispatcher.php line 137

at TraceableEventDispatcher->dispatch('kernel.request', object(GetResponseEvent))in HttpKernel.php line 127

at HttpKernel->handleRaw(object(Request), 1)in HttpKernel.php line 68

at HttpKernel->handle(object(Request), 1, false)in Kernel.php line 200

at Kernel->handle(object(Request), 1, false)in index.php line 82

I´ve searched the foruns, and get results in other languages, but no vaild solid solution, besides a guy solved by restoring a backup from a day before, solutin that im trying to avoid...

The Php version is 7.4 and the hosting service doenst allow below that

Any guesses?

Thanks in advance

**Comment by matks** [comment, marked as answer]
Hello @jpcoke I read https://github.com/doctrine/DoctrineBundle/issues/673

and it says it might be possible to fix the problem by modifying doctrine config, which is inside file app/config/doctrine.yml

See the 'standard' prestashop config file https://github.com/PrestaShop/PrestaShop/blob/8f014136cc306f74da6698a8c02fb73c9fc699b9/app/config/doctrine.yml

According to people in https://github.com/doctrine/DoctrineBundle/issues/673 adding a parameter server_version below works:
So
```
charset: utf8mb4
 ```
 becomes
 ```
charset: utf8mb4
server_version: '5.7'
 ```
 
 Be careful to use same number of whitespaces!
 
 Put 5.7 if you use mysql 5.7, put 5.6 if you use mysql 5.6 ...
